.github/workflows/prlabelling.yaml

```diff
@@ -0,0 +1,16 @@
+name: "Pull Request Labeler"
+run-name: ${{ github.actor }} created Pull Request Labeler
+
+on:
+- pull_request_target
```

Comment:
If I am not mistaken, this event is only produced when a PR from a forked repository is created. 
This is not what we desire here. I think the more general `pull_request` event is better suited for us and we should filter on opening events. Read more about that here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
thanks for the advice
should I change it and then send another pr?

Reply:
You can just leave this PR open and push new commits to your existing branch, then this PR is updated automatically

Reply:
sure,
i'll do that